src/lib/components/SetUpPaymentModal/steps/ConfirmValues.svelte

```diff
@@ -0,0 +1,172 @@
+
+  let actionInFlight = false;
+  let daiBalance: bigint | undefined;
+
+  let topUpAmount = 0;
+  let totalAmount: number = weiToDai(workstream.total);
+
+  $: topUpExceedsBalance =
+    daiBalance < utils.parseUnits(topUpAmount.toString()).toBigInt();
+
+  $: weiPerDay =
+    utils.parseUnits(totalAmount.toString()).toBigInt() /
+    BigInt(workstream.durationDays);
+  $: daiPerDay = currencyFormat(weiPerDay);
+  $: totalWei = weiPerDay * BigInt(workstream.durationDays);
+
+  onMount(async () => {
+    daiBalance = (await drips.getDaiBalance()).toBigInt();
+
+    topUpAmount =
+      daiBalance < totalWei
+        ? weiToDai(daiBalance)
+        : weiToDai(workstream.total.wei);
+  });
+
+  async function setUpPayment() {
+    actionInFlight = true;
+
+    try {
+      const createDripCall = await drips.createDrip(
+        application.creator,
+        {
+          currency: Currency.DAI,
+          wei: weiPerDay / BigInt(86400)
+        },
+        utils.parseUnits(topUpAmount.toString()).toBigInt()
+      );
+
+      const receipt = await createDripCall.tx.wait(1);
+
+      if (receipt.status === 0) {
+        console.error(receipt);
+        return;
+      }
+
+      const activateCall = await workstreamsStore.activateWorkstream(
+        workstream.id,
+        createDripCall.accountId
+      );
+
+      if (activateCall.ok) {
+        dispatch('continue');
+      } else {
+        console.error(activateCall.error);
+      }
+    } catch (e) {
+      console.error(e);
+      actionInFlight = false;
+    }
+  }
+</script>
+
+<div>
+  <span class="emoji">💸</span>
+  <h1>Set up payment stream</h1>
+  <p>Here's the drips config we're gonna set up.</p>
+  <form>
+    <div class="input-with-label">
+      <h4>Streaming to</h4>
+      <Card hoverable={false} style="width: 100%">
+        <div slot="top">
+          <User address={application.creator} />
+        </div>
+      </Card>
+    </div>
+    <div class="inner">
+      <div class="input-with-label">
+        <h4>Total amount</h4>
+        <!-- TODO: wired up to update the stream rate + that's what you're submitting -->
+        <TextInput
+          bind:value={totalAmount}
+          suffix="DAI"
+          style="width: fit-content;"
+          number
+        />
+      </div>
+      <div class="input-with-label rate">
+        <h4 style="color: var(--color-foreground-level-4);">Stream rate</h4>
+        <p>
+          {daiPerDay} DAI / 24h
+        </p>
+      </div>
+    </div>
+    <div class="inner">
+      <div class="input-with-label">
+        <h4>Top up amount</h4>
+        <TextInput
+          bind:value={topUpAmount}
+          suffix="DAI"
+          number
+          style="width: fit-content;"
+          validationState={topUpExceedsBalance
+            ? {
+                type: 'invalid',
+                message: `You only have ${currencyFormat(daiBalance)} DAI`
+              }
+            : { type: 'valid' }}
+        />
+      </div>
+    </div>
```

Comment:
This whole 
`<Form> <div inner> <div input-with-label>` should be it's own component. Since we use it in different places